indra/newview/llgesturemgr.cpp

```diff
@@ -56,6 +56,8 @@
 #include "llappearancemgr.h"
 #include "llgesturelistener.h"
 
+#pragma optimize("", off)
```

Comment:
Why is this here?

Reply:
We should avoid using compiler-specific code, especially in such general areas as this.

Reply:
Removed (there was a temporary hint for debug purposes)